phase2/docker-images/toolbox/scripts/install-basic-systemd-units

```diff
@@ -19,8 +19,8 @@ set -o pipefail
 set -o nounset
```

Comment:
These are part of rebase on Alpine toolbox, but actually are re-implemented in Ansible here. I haven't decided whether to delete bash implementations yet or keep for a bit.